js/ui/form/components/field_item.js

```diff
@@ -99,11 +102,20 @@ export function renderFieldItem({
             $rootElement.addClass(LABEL_HORIZONTAL_ALIGNMENT_CLASS);
         }
 
-        if(item.editorType === 'dxCheckBox' || item.editorType === 'dxSwitch') {
+        if(editorType === 'dxCheckBox' || editorType === 'dxSwitch') {
             eventsEngine.on($label, clickEventName, function() {
                 eventsEngine.trigger($fieldEditorContainer.children(), clickEventName);
             });
         }
+
+        const toggleControlsGroup = ['dxCheckBox', 'dxSwitch', 'dxRadioGroup'];
+        const isToggleControlsGroup = toggleControlsGroup.includes(editorType);
+        const labelAlignment = labelOptions.alignment;
+        const isNotAlignmentRightOrCenter = labelAlignment !== 'right' && labelAlignment !== 'center';
+
+        if(isToggleControlsGroup && labelLocation === 'top' && isNotAlignmentRightOrCenter) {
```

Comment:
Seems like we can just write this way and get rid of a variable above
```suggestion
        if(isToggleControlsGroup && labelLocation === 'top' && labelAlignment === 'left') {
```

Reply:
If we don 't pass the parameter, then it is `undefined`

Reply:
I do not know how to implement this in a different way, if not to make big changes, except to put it in the `isNotAlignmentRightOrCenter` variable

Reply:
> 

And what's the problem?

Reply:
```javascript
const isLabelAlignmentLeft = labelAlignment === 'left' || !labelAlignment;
const isLabelLocationTop = labelLocation === 'top' || !labelLocation;
if(isToggleControlsGroup && isLabelLocationTop && isLabelAlignmentLeft) 
```